tutorials/Binary_Contingency_Scores.ipynb

```diff
@@ -65,7 +67,7 @@
     "- convert the non-binary data into binary data outside of `scores`, then input the binary data into `scores` (in which case, go to step two), OR  \n",
     "- use `scores` to efficiently convert the non-binary data into binary data, using the `scores` \"event operator\" class. \n",
     "\n",
-    "`scores` provides a general class called an \"EventOperator\". `scores` provides in-built event operator types (e.g. greater-than, less-than or many others). Alternatively, users can define their own event operator implementation for use within `scores`.\n",
+    "The `scores` \"event operator\" class provides in-built event operator types (e.g. greater-than, less-than, and many others). Alternatively, users can define their own event operator implementation for use within `scores` (for example, for more complex event definitions involving multiple variables).\n",
```

Comment:
I think \"EventOperator\" may need to remain as is? (But I am not sure, I don't have a coding background).